config/ogdl/src/test/groovy/org/apache/shiro/config/ogdl/ReflectionBuilderTest.groovy

```diff
@@ -36,6 +37,7 @@ import static org.hamcrest.Matchers.*
 /**
  * Unit tests for the {@link org.apache.shiro.config.ogdl.ReflectionBuilder} implementation.
  */
+@Isolated
```

Comment:
nit: add reason for needing to be isolated (we can possibly fix it in the future)

Reply:
haha Dunno.. cause it failed :)